get-terraform-provider-versions/get_terraform_provider_versions/get-tf-provider-versions.py

```diff
@@ -0,0 +1,337 @@
+#!/usr/bin/env python3
```

Comment:
Removed but I think I need to learn more about modules and packages. This line only exists because I actually started my project with one of your source files which contained the shebang.